src/Header/style.scss

```diff
@@ -0,0 +1,3 @@
+.container-home {
+  background: fixed url('../image/Bg2.png') no-repeat center center / cover;
```

Comment:
what does fixed do?

Reply:
it makes a cool effect that page and background doesn`t connected ,when you scroll the page

Reply:
great. thx